voila/paths.py

```diff
@@ -16,7 +16,7 @@
 # if the directory above us contains the following paths, it means we are installed in dev mode (pip install -e .)
 DEV_MODE = os.path.exists(os.path.join(ROOT, '../setup.py')) and os.path.exists(os.path.join(ROOT, '../share'))
 
-notebook_path_regex = r'(.*\.ipynb)'
+notebook_path_regex = r'(.*\.(?:ipynb|py))'
```

Comment:
Wondering if we could get support for other languages for free here (Julia, R and so on)?

Reply:
Indeed, I've added julia in #330, because I could test it. I didn't feel comfortable adding more. I also wonder if we should have a `.*` regex?

Reply:
This regex was added because voila could only handle `.ipynb` files.

But if scripts (in different languages) are going to be supported, it probably make sense to switch back to `.*`. Otherwise we would have to maintain a white list of supported extensions.